custom_components/tahoma/cover.py

```diff
@@ -317,19 +317,41 @@ async def async_my(self, **_):
     @property
     def is_opening(self):
         """Return if the cover is opening or not."""
-        return any(
+        if any(
             execution.get("deviceurl") == self.device.deviceurl
             and execution.get("command_name") in COMMANDS_OPEN + COMMANDS_OPEN_TILT
             for execution in self.coordinator.executions.values()
+        ):
+            return True
+
+        is_moving = self.device.states[CORE_MOVING_STATE].value
```

Comment:
Do we still need the loop through current executions, when we can look at the state as well?

Reply:
I thought about it too. But I don’t know how it will work for the tilt. It’s why I keep it like this.